mmv1/products/cloudidentity/GroupMembership.yaml

```diff
@@ -37,6 +39,8 @@ timeouts:
   update_minutes: 20
   delete_minutes: 20
 custom_code:
+  extra_schema_entry: templates/terraform/extra_schema_entry/cloud_identity_group_membership.tmpl
+  custom_create: templates/terraform/custom_create/cloud_identity_group_membership.go.tmpl
```

Comment:
Instead of an entire custom_create I think this could be accomplished with a pre_create that does the following:

Make the GET request to check if the resource already exists
If no, continue with normal create
If yes, do any necessary setting of computed fields, id, name etc. 

This would make it easier in the future to add new fields to this resource, as the generated create functionality will continue to work as new fields are introduced